metals/src/main/scala/scala/meta/internal/metals/InitializationOptions.scala

```diff
@@ -79,6 +80,7 @@ final case class InitializationOptions(
     disableColorOutput: Option[Boolean],
     doctorVisibilityProvider: Option[Boolean],
     bspStatusBarProvider: Option[String],
+    autoImportBuild: Option[AutoImportBuildKind],
```

Comment:
Shouldn't this be a user rather than a client setting? I think that many users no matter the client don't want to auto import, because they switch to `sbt` right away.

Reply:
Unfortunately so far as I can see Helix, which is one of the main use cases for this I suspect, doesn't seem to have a mechanism for setting user settings

Reply:
Ok digging in a bit deeper looks like helix does support `didChangeConfiguration` so will revert to configuring via user settings